cl/opinion_page/forms.py

```diff
@@ -479,12 +509,299 @@ def save(self) -> OpinionCluster:
+
+        self.order_fields(
+            [
+                "case_title",
+                "docket_number",
+                "publication_date",
+                "date_argued",
+                "date_reargued",
+                "panel",
+                "cite_volume",
+                "cite_reporter",
+                "cite_page",
+                "download_url",
+                "pdf_upload",
+            ]
+        )
+
+    def clean(self) -> dict[str, Any]:
+        super().clean()
+        self.validate_neutral_citation()
+        return self.cleaned_data
+
+
+class TennWorkCompClUploadForm(BaseCourtUploadForm):
+    """
+    Form for Tennessee Court of Workers' Compensation Claims (tennworkcompcl) Upload
+    Portal
+    """
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.add_author_field()
+        self.set_judges_qs()
+        self.add_citation_fields()
+
+        self.fields["cite_reporter"].widget = forms.Select(
+            choices=[("TN WC", "TN WC")],
+            attrs={"class": "form-control", "readonly": "readonly"},
+        )
+
+        self.order_fields(
+            [
+                "case_title",
+                "docket_number",
+                "publication_date",
+                "lead_author",
+                "cite_volume",
+                "cite_reporter",
+                "cite_page",
+                "download_url",
+                "pdf_upload",
+            ]
+        )
+
+    def clean(self) -> dict[str, Any]:
+        super().clean()
+        self.validate_neutral_citation()
+        return self.cleaned_data
+
+
+class TennWorkCompAppUploadForm(BaseCourtUploadForm):
+    """
+    Form for Tennessee Workers' Compensation Appeals Board (tennworkcompapp) Upload
+    Portal
+    """
+
+    second_judge = forms.ModelChoiceField(
+        queryset=Person.objects.none(),
+        required=False,
+        label="Second Panelist",
+        widget=forms.Select(attrs={"class": "form-control"}),
+    )
+    third_judge = forms.ModelChoiceField(
+        queryset=Person.objects.none(),
+        required=False,
+        label="Third Panelist",
+        widget=forms.Select(attrs={"class": "form-control"}),
+    )
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.add_author_field()
+        self.set_judges_qs()
+        self.add_citation_fields()
+
+        self.fields["cite_reporter"].widget = forms.Select(
+            choices=[("TN WC App.", "TN WC App.")],
+            attrs={"class": "form-control", "readonly": "readonly"},
+        )
+
+        self.order_fields(
+            [
+                "case_title",
+                "docket_number",
+                "publication_date",
+                "lead_author",
+                "second_judge",
+                "third_judge",
+                "cite_volume",
+                "cite_reporter",
+                "cite_page",
+                "download_url",
```

Comment:
see above- no download url exists right?

Reply:
right, I already removed it